app/logic/pager.rb

```diff
@@ -0,0 +1,18 @@
+class Pager
```

Comment:
@vlasiak, Also is this class a pager or some parameters/attributes of a pager that are adjusted after creation event? Let's think on proper name.